tests/openwisp2/settings.py

```diff
@@ -196,6 +196,8 @@
     connection_index = INSTALLED_APPS.index('openwisp_controller.connection')
     INSTALLED_APPS.remove('openwisp_controller.connection')
     INSTALLED_APPS.insert(connection_index, 'openwisp2.sample_connection')
+    INSTALLED_APPS.remove('openwisp_users')
```

Comment:
Removed `openwisp_users` from installed apps when **SAMPLE_APP** is true

Reply:
This should not be required. The tests related to extending openwisp-users should be already present in the sample app of openwisp-users. 

@devkapilbansal why did you make these changes?

Reply:
I did it as done in `openwisp-radius` here :point_down: 
https://github.com/openwisp/openwisp-radius/blob/e182417a10e4f0a0572047ab08181fb057163292/tests/openwisp2/settings.py#L244

Also, if I don't remove the `openwisp_users` from `INSTALLED_APPS` it gives rise to reverse accessor clashes between `openwisp-users` and `sample-users`

Reply:
I believe you don't need `sample-users` here. But let's get a confirmation from @atb00ker or @nemesisdesign before proceeding.